packages/browser/package.json

```diff
@@ -13,35 +13,24 @@
 	"module": "lib/index.es.js",
 	"peerDependencies": {
 		"react": "^16.5.1",
-		"react-dom": "^16.5.1"
-	},
-	"dependencies": {
+		"react-dom": "^16.5.1",
 		"@appbaseio/designkit": "^0.9.1",
 		"@appbaseio/reactivesearch": "^3.0.2",
-		"@divyanshu013/media": "^1.0.0",
 		"antd": "^3.10.4",
 		"appbase-js": "^4.0.2-beta.2",
 		"brace": "^0.11.1",
 		"codesandbox": "1.3.5",
```

Comment:
I don't think we need `codesandbox` or any packages related to Search Preview here, they should be part of `dejavu-main`.

You can try removing packages listed here and not used by `browser`
https://github.com/appbaseio/batteries